src/main/scala/no/ndla/learningpathapi/controller/LearningpathController.scala

```diff
@@ -70,6 +70,18 @@ trait LearningpathController {
         responseMessages(response400, response500)
         authorizations "oauth2")
 
+    val getLearningpathsPost =
+      (apiOperation[List[SearchResult]]("getAllArticlesPost")
```

Comment:
Siden endepunktet er `/search` så ville jeg heller kallt denne f.eks `searchArticles` i stedet for `getAllArticlesPost`. POST er redundant. Det ser vi på http-metoden.